toolkit/themes/shared/design-system/README.design-tokens-JSON.stories.md

```diff
@@ -0,0 +1,60 @@
+# Design tokens
```

Comment:
We need to change the file name to lower case otherwise the file doesn't show up